paypal/driver.py

```diff
@@ -117,6 +120,29 @@ def _get_value_from_qs(self, qs, value):
         else:
             return raw
 
+    def _nvp_request(self, parameters):
+        """Convenience method for making requests to keep driver DRY
+        :param parameters: parameters to send in the request
+        :type parameters: dictionary
+
+        :returns: value of the ACK parameter in the response, can be
+        "Success", "SuccessWithWarning", "Failure", "FailureWithWarning"
+        :rtype: string
+        """
+        query_string = self.signature + urllib.urlencode(parameters or {})
+        response = urllib2.urlopen(self.NVP_API_ENDPOINT, query_string).read()
+        response_tokens = {}
+        for token in response.split('&'):
```

Comment:
Don't parse query strings like this. Use python features:
https://docs.python.org/2/library/urlparse.html#urlparse.parse_qs